hrjobcontract/tests/phpunit/CRM/Hrjobcontract/Import/Parser/ApiTest.php

```diff
@@ -47,6 +53,22 @@ function setUp() {
     ];
   }
 
+  function createPayScale() {
```

Comment:
Ok, so the real reason teardowns are not required is this class implements TransactionalInterface interface, which handles all clean-up at the end of the test.